lib/perl/Genome/WorkflowBuilder/Converge.pm

```diff
@@ -73,4 +74,37 @@ sub notify_output_link {
     return;
 }
 
+sub get_ptero_builder_task {
+    require Ptero::Builder::Detail::Workflow::Task;
```

Comment:
Cool, that makes sense to me - I'm fine leaving it as is for now, and moving it to the top when we migrate fully!